README.md

```diff
@@ -39,7 +39,9 @@ $ obom show -f ./examples/SPDXJSONExample-v2.3.spdx.json
 Document Name:         SPDX-Tools-v2.0
 DataLicense:           CC0-1.0
 Document Namespace:    http://spdx.org/spdxdocs/spdx-example-444504E0-4F89-41D3-9A0C-0305E82C3301
-SPDX Version:          SPDX-2.3
+SPDX Version:          SPDX-2.3 # NOTE This is the default version for all spdx you have to comment out line 76 
+                                # from go\pkg\mod\github.com\spdx\tools-golang@v0.5.0\spdx\v2\v2_3\document.go
```

Comment:
is this a known bug? should we move this into a comment into the code instead of the README?